HTML/mockup-corporate/js/task3.js

```diff
@@ -0,0 +1,48 @@
+import {elements} from './elements.js';
+import {patterns} from './patterns.js';
+
+const inputNameArr = [
+  elements.nameInput,
+  elements.emailInput,
+  elements.passwordInput,
+];
+
+const patternArr = [patterns.name, patterns.email, patterns.password];
+
+const validator = (input, pattern) => {
+  let inputValue = input.value;
+  let validation = pattern.test(inputValue);
+
+  input.classList.remove('input--invalid');
+
+  if (!validation) {
+    input.classList.add('input--invalid');
+    return false;
+  } else {
+    return true;
+  }
+};
+
+const validArr = [false, false, false];
+
+inputNameArr.forEach((el, ind) => {
+  el.addEventListener('input', () => {
+    let validValue = validator(inputNameArr[ind], patternArr[ind]);
+    validArr[ind] = validValue;
+
+    elements.btn.classList.remove('btn--disabled');
```

Comment:
there is disabled attribute better to use one, styles for disabled button make this button almost invisible, what is not good ui